2018 Robot/src/org/usfirst/frc/team4999/commands/intake/ShootIntake.java

```diff
@@ -0,0 +1,49 @@
+package org.usfirst.frc.team4999.commands.intake;
+
+import org.usfirst.frc.team4999.robot.Robot;
+import org.usfirst.frc.team4999.robot.subsystems.Intake;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class ShootIntake extends Command {
+	
+	private Intake intake = Robot.intake;
+	
+	private static final double SHOOT_SPEED = 1;
+	private static final double SHOOT_TIME = 2;
```

Comment:
Might be ok with just a 1 second shot.  Typical exit time for the cube seems to be in the quarter second range.